scripts/find_rust_repos.py

```diff
@@ -0,0 +1,120 @@
+"""
+Finds new Rust repo(s) and checks if the repo(s) meet requirements
```

Comment:
since we can pass command line arguments to `find_repos.py` do we need an extra file to find rust repos?

Reply:
Actually we do not need to. Just change variable content in main function of find_repo.py, we could find rust repos.

Reply:
Should I delete the file then? I'm going to commit some new changes soon anyway.

Reply:
Deleting the file is OK, and please add an example command to run rust's `checks_list` in README.md